### PR TITLE
Hw03

### PR DESCRIPTION
**Pull request by samruddhis**
@Han-Lee93 @Naidantu

### CODE REVIEW

File: hw03/hw03.Rmd [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+            intel_sd = sd(intel_pomp),
+            intel_median = median(intel_pomp),
+            intel_min = min(intel_pomp),
+            intel_max = max(intel_pomp),
+            open_mean = mean(open_pomp),
+            open_sd = sd(open_pomp),
+            open_median = median(open_pomp),
+            open_min = min(open_pomp),
+            open_max = max(open_pomp))
+
```
Comment (Han-Lee93) [inline hw03/hw03.Rmd:218]: You can also shorten this by using the across function as such:

summarize(across(c("column1":"column2")

File: hw03/hw03.Rmd [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+      group = student,
+      color = student,
+      fill = student) +
+  geom_jitter()
+```
+     
+  6. Compare **green reputation** for students and non-students using a **rainfall plot** (bar + density + data points).
+  
+```{r}
+
```
Comment (Han-Lee93) [inline hw03/hw03.Rmd]: Missing Q6?

**Review by Han-Lee93 — COMMENTED** [review]
----------------------------------------------------
Topic                       | Excellent | Satisfactory | Needs Work |
----------------------------|-----|-----|-----|
**Coding style**            | :heavy_check_mark: |  |  |
**Coding strategy**         | :heavy_check_mark: |  |  |
**Code commenting**          |  |  | :heavy_check_mark: |
**Presentation: figure**    |   | | :heavy_check_mark: |
**Achievement, creativity** | :heavy_check_mark: |  |  |
**Ease of access**          | :heavy_check_mark: |  |  |
**Neatness**          | :heavy_check_mark: |  |  |

Remarks:
Hard to review the work without the md file to look at the graphs, but I did not see any problems worth noting outside of Q4:7 were not answered. Code could also use some commenting so that reviewers can understand the code with more ease.

File: hw03/hw03.Rmd [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+    .id = id,
+    green_total = mean(c(green1, green2r, green3, green4, green5), na.rm = TRUE),
+    comp_total = mean(c(comp1, comp2, comp3, comp4, comp5, comp6r, comp7r, comp8r, comp9r, comp10r), na.rm = TRUE),
+    intel_total = mean(c(intel1, intel2, intel3, intel4, intel5, intel6, intel7r, intel8r, intel9r, intel10r), na.rm = TRUE),
+    open_total = mean(c(open1, open2, open3, open4, open5, open6, open7r, open8r, open9r, open10r), na.rm = TRUE))
+```
+
+  3. Rescale the variables so that they go from 0-100 instead of the original range.
+     Name the recaled variables `*_pomp`.
+
```
Comment (Naidantu) [inline hw03/hw03.Rmd:84]: I think this question asks us to change the range of the total scores, not each item

File: hw03/hw03.Rmd [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+  4. Make plots that illustrate the distributions of the 4 POMP-scored variables.
+  
+```{r}
+ggplot(green_data) +
+  aes(x = green_pomp) +
+  geom_bar() +
+  theme_bw()
+
```
Comment (Naidantu) [inline hw03/hw03.Rmd:115]: the bar graph is for categorical data, histogram might be a better option for the total scores

File: hw03/hw03.Rmd [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+            open_sd = sd(open_pomp),
+            open_median = median(open_pomp),
+            open_min = min(open_pomp),
+            open_max = max(open_pomp))
+
+print(summary_green)
+```
+  
+  
+In your assignment, prepare an RMarkdown file that includes both the coding and (for 4-7) narrative descriptions of your results.
```
Comment (Naidantu) [inline hw03/hw03.Rmd]: some narrative descriptions are needed for q4-7. Overall nice job!

**Pull request merged**